wpilib/wpilib/sendablechooser.py

```diff
@@ -5,8 +5,12 @@
 # must be accompanied by the FIRST BSD license file in the root directory of
 # the project.
 # ----------------------------------------------------------------------------
+from typing import TypeVar
+
+V = TypeVar("V", covariant=True)
```

Comment:
FYI, you're still just using this as a convoluted way of saying Any.

Reply:
Yeah if I said `Any` that wouldn't relate the classes

Reply:
Oh I guess you can add multiple types to a chooser so ok.

Reply:
You had no relationship between them anyway?

Reply:
Yeah, before if I put in type V, I was gonna get a type V back

Reply:
V was a free variable. i.e. in each signature, V did not relate to any V in any other signature.